doc/source/serve/tutorials/batch.md

```diff
@@ -15,16 +15,16 @@ For _offline_ batch inference with large datasets, see [batch inference with Ray
 
 
 ## Define the Deployment
-Open a new Python file called `tutorial_batch.py`. First, let's import Ray Serve and some other helpers.
+Open a new Python file called `tutorial_batch.py`. First, import Ray Serve and some other helpers.
 
 ```{literalinclude} ../doc_code/tutorial_batch.py
 :end-before: __doc_import_end__
 :start-after: __doc_import_begin__
 ```
 
 You can use the `@serve.batch` decorator to annotate a function or a method.
-This annotation will automatically cause calls to the function to be batched together.
-The function must handle a list of objects and will be called with a single object.
+This annotation automatically causes Ray Serve to batch calls to the function together.
+The function must handle a list of objects and Ray Serve calls it with a single object.
```

Comment:
```suggestion
The function is called with a single object but must take in and handle a list of objects. This allows Ray Serve to batch together multiple function and pass the calls' inputs into the function as a list.
```

Reply:
TY! Who calls the function? I'm hoping there's a way to write the first sentence in active voice.